olp-cpp-sdk-core/include/olp/core/http/NetworkTypes.h

```diff
@@ -128,5 +129,9 @@ class SendOutcome final {
  */
 CORE_API std::string ErrorCodeToString(ErrorCode code);
 
+/**
+ * @brief Type alias for HTTP headers.
+ */
+using HeadersType = std::multimap<std::string, std::string>;
```

Comment:
Why multimap? It is basically the same as `vector<pair<string, string>>`, except it consumes additional memory and it keeps elements sorted. Do we need this? Is there any benefit? Is it worth it to change API over it?

Reply:
**TL;DR**
Even though the `vector` of pairs is not as usable as a map, it also okay with me.

**Detailed answer.**
That's my fault. Working with `OlpClient::CallApi` got me thinking that `multipmaps` are the way to do the headers thing around here. So I confused Liubov by asking her to be consistent with the `CallApi` and extract a separate header type using `multimap`.

I also consider maps to be the best choice for storing headers as it's easier to do things such as:
```
// Pseudo-code starts
headers["Authorization"] = "Bearer h1.blah";
client.Get("http://example.com", headers);
// Pseudo-code ends
```

Reply:
multimap misses the operator[] as far as I know as it does not have unique keys.

Reply:
That's sad. But on the bright side, it has `find(const Key& key)` which is one step closer to the `std::map::[]` than a vector.

Reply:
Even if we leave std::vector, it seems to be good to have one type alias in NetworkTypes.h. What do you think?

Reply:
+1 to having `using HeadersType = std::vector<std::pair<std::string, std::string>>`

Reply:
done